src/Microsoft.Dnx.Host.Mono/EntryPoint.cs

```diff
@@ -76,7 +76,8 @@ public static int Main(string[] arguments)
 
         return RuntimeBootstrapper.Execute(arguments,
             // NOTE(anurse): Mono is always "dnx451" (for now).
-            new FrameworkName("DNX", new Version(4, 5, 1)));
+            new FrameworkName("DNX", new Version(4, 5, 1)),
+            null);
```

Comment:
No. At the moment I am working just on passing the struct and not on using what is passed so null is fine here.